lib/cell.rb

```diff
@@ -28,7 +28,7 @@ def fired_upon?
   end
 
   def render(conditional = false)
-    if conditional == true
+    if conditional == true && empty? == false
```

Comment:
Adds functionality for board.render to ensure only printing "S" where a ship is.